save-common/src/commonMain/kotlin/org/cqfn/save/core/utils/ProcessBuilder.kt

```diff
@@ -102,7 +102,7 @@ class ProcessBuilder(private val useInternalRedirections: Boolean, private val f
         fs.deleteRecursively(tmpDir)
         logDebug("Removed temp directory $tmpDir")
         if (stderr.isNotEmpty()) {
-            logWarn(stderr.joinToString("\n"))
+            logDebug(stderr.joinToString("\n"))
```

Comment:
The following errors occured after executing: <execCmd>

Reply:
Fixed